src/plugins/intel_cpu/src/utils/jit_kernel.hpp

```diff
@@ -668,8 +668,12 @@ variable_base<T>::variable_base(variable_base && rhs)
 
 template<typename T>
 variable_base<T>::~variable_base() {
-    if (_manage_lifetime)
-        kernel.free(_reg);
+    try {
+        if (_manage_lifetime)
+            kernel.free(_reg);
+    } catch (std::exception &e) {
+        std::cerr << e.what() << std::endl;
```

Comment:
Not sure this is good behavior. What's the coverity error?

Reply:
In MKLDNNPlugin::​internal::​variable_base<unsigned long const>::​~variable_base(): A C++ exception is thrown but never caught. We must catch the error in the destructor.

Reply:
Still think just printing error message in the stream is not a good way of handling. @vladislav-volkov Maybe you can suggest smt.

Reply:
This issue should be fixed on master. Could you please rebase to latest version?
Destructors should never throw exceptions. So we can only catch them and suppress.